src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2194,7 +2194,7 @@
     <!-- Whether receiver will need to be cloned during emit (only valid before lowering) -->
     <Field Name="InitialBindingReceiverIsSubjectToCloning" Type="ThreeState"/>
     <Field Name="PropertySymbol" Type="PropertySymbol"/>
-    <!-- The property accessor referenced if the property access should use the backing field directly instead. -->
+    <!-- The property accessor kind, set for properties with synthesized backing field only. -->
     <Field Name="UseBackingField" Type="AccessorKind" />
```

Comment:
I'm not 100% happy with this field name, but I don't have any suggestions for a better one. It feels like this name is saying "this property access should be replaced with a use of the backing field".

Reply:
Renamed to `BoundPropertyAccess.AutoPropertyAccessorKind`.